### PR TITLE
Bug/150/ui bugs

### PR DESCRIPTION
**Pull request by xcaverxiii**
fix some ui bugs
move styles into own folder

related issue #150

### CODE REVIEW

**Comment by EeveesEyes** [comment]
There is a problem with the default value of the collapse element (more). Artworks with a `metaNumber > 10` should have a collapsed 'more' section.

File: app/src/app/pages/artwork/artwork.component.html [hunk 1]
```diff
@@ -107,17 +107,20 @@ <h2 class="mb-3 first-letter">{{ artwork?.label }}</h2>
           artwork?.materials?.length ||
           artwork?.motifs?.length
         ">
-        <h4 class="mt-2 more" href="#collapseArtwork" (click)="toggleDetails()">
+        <h4 class="mt-2 more" data-toggle="collapse" href="#collapseDetails" (click)="toggleDetails()">
```
Comment (EeveesEyes) [inline app/src/app/pages/artwork/artwork.component.html]: Here is a problem with the default value of the collapse element (more). Artworks with a metaNumber > 10 should have a collapsed 'more' section.

Comment (xcaverxiii) [reply app/src/app/pages/artwork/artwork.component.html]: Do you know some solution how to set the initial value of the collapse state?
This is not working, or i doing something wrong...

`<div class="collapse" [ngbCollapse]="!collapseDetails" id="collapseDetails">`
` <div class="collapse" collapse="!collapseDetails" id="collapseDetails">`
`<div class="collapse" [ngClass]="{'in': !collapseDetails}" id="collapseDetails">`

Comment (EeveesEyes) [reply app/src/app/pages/artwork/artwork.component.html]: I tried yesterday, but i honestly dont know exactly how the default state is applied there. Has to be an easy fix tho.

Comment (xcaverxiii) [reply app/src/app/pages/artwork/artwork.component.html]: I´m so frustrated with angular -.- 
This framework is so unsmart.... Maybe i find a solution till this week.

**Comment by xcaverxiii** [comment]
@EeveesEyes 
I solved the problem. Can you now do another review?

**Pull request merged**